src/bun.js/bindings/bindings.zig

```diff
@@ -3542,6 +3542,18 @@ pub const JSValue = enum(JSValueReprInt) {
         return this.jsType();
     }
 
+    extern fn JSC__jsTypeStringForValue(globalObject: *JSGlobalObject, value: JSValue) *JSC.JSString;
+
+    pub fn jsTypeString(this: JSValue, globalObject: *JSGlobalObject) *JSC.JSString {
+        return JSC__jsTypeStringForValue(globalObject, this);
+    }
+
+    extern fn JSC__JSValue__constructEmptyObject(globalObject: *JSGlobalObject, prototype: [*c]JSC.JSObject, len: usize) JSValue;
```

Comment:
I needed this to create an object with null prototype, `createEmptyObject` creates it with the `Object` prototype

Reply:
ah, did not realize the zig `null` turned this into a null prototype. maybe the method name should simply say it's going to have a null prototype, like `createObjectWithNullProto`, but that is a bit verbose. i just suspect `createEmptyObject` and `constructEmptyObject` are going to be very confusing together in the same namespace.

Reply:
I decided to go with the same name as the JavaScriptCore equivalent in C++, and leave it generic enough so it can be also used to create objects with an specific prototype different than Object (it's something I'm using for other wip stuff).

I'd rather leave it as it, but I will change if needed